static/js/containers/pages/CheckoutPage_test.js

```diff
@@ -203,7 +201,7 @@ describe("CheckoutPage", () => {
           body: {
             items: [
               {
-                product_id: basketItem.product_id.toString(),
```

Comment:
Why the `toString` removal?

Reply:
It's for fixing test cases if I pass it all of the test cases started to fail.

Reply:
Please investigate the issue properly and fix that rather than simply adding the string conversion directly.